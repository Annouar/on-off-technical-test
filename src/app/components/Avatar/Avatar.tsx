import React from 'react';
import styled from 'styled-components/macro';

const Circle = styled.div`
  background-color: #ccc;
  border-radius: 50%;
  height: 4rem;
  text-align: center;
  width: 4rem;
`;

const Initials = styled.span`
  font-size: 2rem;
  line-height: 1;
  position: relative;
  top: 1rem;
`;

export const Avatar = () => (
  <Circle>
    <Initials>#</Initials>
  </Circle>
);
