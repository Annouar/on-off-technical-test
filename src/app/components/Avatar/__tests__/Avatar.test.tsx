import React from 'react';
import { render } from '@testing-library/react';

import { Avatar } from '../Avatar';

describe('<Avatar  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<Avatar />);

    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
