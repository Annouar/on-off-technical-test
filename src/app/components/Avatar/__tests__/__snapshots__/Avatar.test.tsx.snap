// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Avatar  /> should match snapshot 1`] = `
.c0 {
  background-color: #ccc;
  border-radius: 50%;
  height: 4rem;
  text-align: center;
  width: 4rem;
}

.c1 {
  font-size: 2rem;
  line-height: 1;
  position: relative;
  top: 1rem;
}

<div
  class="c0"
>
  <span
    class="c1"
  >
    #
  </span>
</div>
`;
